test/evergreen.yml

```diff
@@ -3034,138 +3053,335 @@ tasks:
+
+  - name: perf-test-parallel-pop-stress
+    tags: ["stress-perf"]
+    depends_on:
+      - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - func: "run-perf-test"
+        vars:
+          perf-test-name: parallel-pop-stress
+          maxruns: 1
+      - func: "upload-perf-test-stats"
+        vars:
+          perf-test-name: parallel-pop-stress
+
+  - name: perf-test-update-grow-stress
+    tags: ["stress-perf"]
+    depends_on:
+      - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - func: "run-perf-test"
+        vars:
+          perf-test-name: update-grow-stress
+          maxruns: 1
+          wtarg: "-a ../../../bench/wtperf/runners/update-stress.json"
+      - func: "upload-perf-test-stats"
+        vars:
+          perf-test-name: update-grow-stress
+
+  - name: perf-test-update-shrink-stress
+    tags: ["stress-perf"]
+    depends_on:
+      - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - func: "run-perf-test"
+        vars:
+          perf-test-name: update-shrink-stress
+          maxruns: 1
+          wtarg: "-a ../../../bench/wtperf/runners/update-stress.json"
+      - func: "upload-perf-test-stats"
+        vars:
+          perf-test-name: update-shrink-stress
+
+  - name: perf-test-update-delta-mix1
+    tags: ["stress-perf"]
+    depends_on:
+      - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - func: "run-perf-test"
+        vars:
+          perf-test-name: update-delta-mix1
+          maxruns: 1
+          wtarg: "-a ../../../bench/wtperf/runners/update-stress.json"
+      - func: "upload-perf-test-stats"
+        vars:
+          perf-test-name: update-delta-mix1
+
+  - name: perf-test-update-delta-mix2
+    tags: ["stress-perf"]
+    depends_on:
+      - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - func: "run-perf-test"
+        vars:
+          perf-test-name: update-delta-mix2
+          maxruns: 1
+          wtarg: "-a ../../../bench/wtperf/runners/update-stress.json"
+      - func: "upload-perf-test-stats"
+        vars:
+          perf-test-name: update-delta-mix2
+
+  - name: perf-test-update-delta-mix3
+    tags: ["stress-perf"]
+    depends_on:
+      - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - func: "run-perf-test"
+        vars:
+          perf-test-name: update-delta-mix3
+          maxruns: 1
+          wtarg: "-a ../../../bench/wtperf/runners/update-stress.json"
+      - func: "upload-perf-test-stats"
+        vars:
+          perf-test-name: update-delta-mix3
+
+  - name: perf-test-multi-btree-zipfian
+    tags: ["stress-perf"]
+    depends_on:
+      - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - func: "run-perf-test"
+        vars:
+          perf-test-name: multi-btree-zipfian-populate
+          maxruns: 1
+      - func: "run-perf-test"
+        vars:
+          perf-test-name: multi-btree-zipfian-workload
```

Comment:
Does this overwrite the `atlas_out.json` and `evergreen_out.json` from `multi-btree-zipfian-populate` above?

Reply:
Yup, it overwrites `atlas_out.json` and `evergreen_out.json`, as we do not require to plot the data generated by `multi-btree-zipfian-populate`
Basically, the `multi-btree-zipfian-workload` has to run on the same directory created by `multi-btree-zipfian-populate`. Hence, it is in the same task. 
Also, in Jenkins the graph is plotted only for the read count generated from the `multi-btree-zipfian-workload`